main.go

```diff
@@ -2,16 +2,26 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"os"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
-	"os"
+)
+
+var (
+	VERSION   = "v0.11.0"
```

Comment:
@wlan0  I'm worried about VERSION becoming out of date because it requires a manual step to update. Can we make do with jsut the git commit? Seems like it should be enogh.